### ISSUE TITLE
Since update to 1.16.0, bullet and numbered lists have major display issue

### ISSUE THREAD

**Issue by WizWareTech** [humhub/wiki#335]
Here is what it looks like in the Edit window:

![image](https://github.com/humhub/wiki/assets/13114/4926048e-531d-45c1-b099-c23c84bfe765)

Here is what it looks like when displayed:

![image](https://github.com/humhub/wiki/assets/13114/0ba0a18b-de7f-4cc6-8898-344dfba42635)

The problem exists even with HumHub theme. Everything was fine prior to updating. The other issue I noticed that when editing and clicking on the right side of a sub-item to move the cursor to that area, it does not respond to the click unless you click just to the immediate right of the bullet. This behavior did not exist before.

I found that if I turned this off in dev mode, it fixes the display issue:
![image](https://github.com/humhub/wiki/assets/13114/db0b80d9-fc0f-4d52-a1b8-de2f6913166e)

What is the right way to permanently fix this?

Another issue that DID exist before and still exists is this: When the cursor is at a particular location that you want to change to bullet/numbered list, and you click on bullet/numbered list icon in toolbar, the focus changes to toolbar and you must click again on the line you just changed to bullet/numbered list in order to keep typing the list. It would be nice if the focus did not change, and the cursor stayed where you clicked on toolbar.

HumHub Version: 1.16.0
PHP: 8.3.4

**Comment by marc-farre** [comment]
@luke- I also have this issue on latest develop branch, even after rebuilding the community theme.

**Comment by luke-** [comment]
@yurabakhtin Didn't you fix something here recently?

**Comment by yurabakhtin** [comment]
> @yurabakhtin Didn't you fix something here recently?

@luke- Yes, it was broken [here](https://github.com/humhub/humhub/pull/7030) by me.

I have fixed in new PR https://github.com/humhub/humhub/pull/7089:

before fix:
![before-fix](https://github.com/humhub/wiki/assets/6995524/541a0882-ad13-48c8-9c92-f304fe9656a2)

after fix:
![after-fix](https://github.com/humhub/wiki/assets/6995524/bee5b15f-1471-4d11-8d88-bf7b370d45b4)

**Comment by yurabakhtin** [comment]
@WizWareTech 

> Another issue that DID exist before and still exists is this: When the cursor is at a particular location that you want to change to bullet/numbered list, and you click on bullet/numbered list icon in toolbar, the focus changes to toolbar and you must click again on the line you just changed to bullet/numbered list in order to keep typing the list. It would be nice if the focus did not change, and the cursor stayed where you clicked on toolbar.

Do you mean the issue so the list buttons are hidden when we click to a list element like on my video below?


https://github.com/humhub/wiki/assets/6995524/deb69744-ca70-4225-8469-447e7f743c9b

**Comment by luke-** [comment]
@yurabakhtin Thanks. 

Fix will be avaiable with HumHub 1.16.1

**Comment by WizWareTech** [comment]
> @WizWareTech
> 
> > Another issue that DID exist before and still exists is this: When the cursor is at a particular location that you want to change to bullet/numbered list, and you click on bullet/numbered list icon in toolbar, the focus changes to toolbar and you must click again on the line you just changed to bullet/numbered list in order to keep typing the list. It would be nice if the focus did not change, and the cursor stayed where you clicked on toolbar.
> 
> Do you mean the issue so the list buttons are hidden when we click to a list element like on my video below?
> list.mp4

No, I'm saying that when you click on the icon to enable list or indent, the cursor moves away from the line that it was on before you click in the toolbar. Then you have to scroll down and click on that line again to keep typing on that same line.

**Comment by yurabakhtin** [comment]
> No, I'm saying that when you click on the icon to enable list or indent, the cursor moves away from the line that it was on before you click in the toolbar. Then you have to scroll down and click on that line again to keep typing on that same line.

@WizWareTech Can you find the issue on my video below?

https://github.com/humhub/wiki/assets/6995524/13d7b385-88d3-4a1b-b7cf-28b8e7febd04

**Comment by WizWareTech** [comment]
> > No, I'm saying that when you click on the icon to enable list or indent, the cursor moves away from the line that it was on before you click in the toolbar. Then you have to scroll down and click on that line again to keep typing on that same line.
> 
> @WizWareTech Can you find the issue on my video below?
> check-list.mp4

Yes, I see it there. Is there a way to keep cursor in same location so when clicking the list icon in toolbar, you can immediately start typing without having to click location to put cursor again before starting to type?

**Comment by yurabakhtin** [comment]
> Yes, I see it there. Is there a way to keep cursor in same location so when clicking the list icon in toolbar, you can immediately start typing without having to click location to put cursor again before starting to type?

@WizWareTech Thanks for the response, I got it, yes it seems only the list buttons have the unfocus issue, others doesn't move a cursor focus from the text editor to the toolbar. Ok, we will investigate this.

**Comment by yurabakhtin** [comment]
@luke- @WizWareTech PR https://github.com/humhub/humhub-prosemirror/pull/148 for fix [the unfocus issue](https://github.com/humhub/wiki/issues/335#issuecomment-2197208757).
I didn't find a reason why it was implemented [here](https://github.com/humhub/humhub-prosemirror/commit/78ae5ab82916b681e25a0ef69899770085acba6c#diff-c6c1259fef95238485835009fd16b4732f6d3e5c621c108caffcc1424ae49d5dR107-R109).
